chart/flux/README.md

```diff
@@ -185,43 +185,51 @@ The following tables lists the configurable parameters of the Weave Flux chart a
 | ------------------------------- | ------------------------------------------ | ---------------------------------------------------------- |
 | `image.repository` | Image repository | `quay.io/weaveworks/flux`
 | `image.tag` | Image tag | `<VERSION>`
+| `replicaCount` | Number of flux pods to deploy | `1`
 | `image.pullPolicy` | Image pull policy | `IfNotPresent`
-| `resources` | CPU/memory resource requests/limits for Flux | None
-| `token` | Weave Cloud service token | None
-| `extraEnvs` | Extra environment variables for the Flux pod | `[]`
+| `resources` | CPU/memory resource requests/limits for the flux deployment | `{ requests: { cpu: 50m, memory: 64Mi }}`
```

Comment:
Please review this part `{ requests: { cpu: 50m, memory: 64Mi }}`
I'm not sure it's correct - not sure of how exactly to write that when it's inline yaml.

Reply:
Add two rows in the table, one for each setting `resources.cpu` and `resources.memory`

Reply:
changed, please take another look.

Reply:
You should remove the `resources` row now.